test/fixtures/rules/css/test.css

```diff
@@ -0,0 +1,7 @@
+.usedClass{
```

Comment:
This file as some CSS linting errors. We could have added https://github.com/moxystudio/stylelint-config-moxy to detect those errors earlier, but it's unnecessary. See my comments below.

Reply:
fixed the errors